extensions/amp-truncate-text/1.0/storybook/Basic.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as Preact from '../../../../src/preact';
+import {TruncateText} from '../truncate-text';
+import {text, withKnobs} from '@storybook/addon-knobs';
+import {withA11y} from '@storybook/addon-a11y';
+
+export default {
+  title: 'TruncateText',
+  component: TruncateText,
+  decorators: [withKnobs, withA11y],
+};
+
+export const SimpleTruncatedText = () => {
+  const width = text('width', '20em');
+  const height = text('height', '4em');
+
+  return (
+    <TruncateText
+      className="amp-truncate-text"
+      layout="fixed"
```

Comment:
`layout` is an AMP construct, please remove

Reply:
Done